conf/modules.config

```diff
@@ -240,6 +240,10 @@ process { // SUBWORKFLOW: GFF_MERGE_CLEANUP
         ext.prefix = { "${meta.id}.liftoff.braker" }
     }
 
+    withName: '.*:GFF_MERGE_CLEANUP:FILTER_BY_ORF_SIZE' {
+        ext.args = params.filter_genes_by_aa_length ? "--no-pseudo --keep-genes -C -l ${params.filter_genes_by_aa_length * 3}" : ''
```

Comment:
@rosscrowhurst 

params.filter_genes_by_aa_length is multiplied by 3 to get the CDS length for the `-l` parameter.